doc/api.md

```diff
@@ -134,6 +138,51 @@ _This command signs a Solana Off-Chain Message after having the user validate th
 | ------------- | :------: |
 | Signature     |    64    |
 
+### GET CHALLENGE
+
+#### Description
+
+_This command returns a 32-bit challenge generated by the app
+
+##### Command
+
+| _CLA_ | _INS_ | _P1_ | _P2_ |   _Lc_   |     _Le_ |
+| ----- | :---: | ---: | ---- | :------: | -------: |
+| E0    |  20   |   00 | 00   | 00       | N/A      |
+
+##### Input data
+
+N/A
+
+##### Output data
+
+| _Description_ | _Length_ |
+| ------------- | :------: |
+| Challenge     |    4     |
+
+### PROVIDE TRUSTED NAME TLV DESCRIPTOR
+
+#### Description
+
+_This command provides a [Solana Trusted Name TLV descriptor](https://ledgerhq.atlassian.net/wiki/spaces/BE/pages/5123145859/Solana+token+account+ownership)
+
+##### Command
+
+| _CLA_ | _INS_ | _P1_ | _P2_ |   _Lc_   |     _Le_ |
+| ----- | :---: | ---: | ---- | :------: | -------: |
+| E0    |  20   |   00 | 00   | F7 (max) | variable |
```

Comment:
Hey I think that I spotted a small mistake in the doc here
```suggestion
| E0    |  21   |   00 | 00   | F7 (max) | variable |
```

Reply:
Fixed